src/Parser/Monk/Windwalker/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,210 @@
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.CHI_WAVE_TALENT,
+            when: combatant.hasTalent(SPELLS.CHI_WAVE_TALENT.id),
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: 'Use your procs and short CDs',
+      description: <Wrapper>Make sure to use your procs and spells at the correct time. Wasting <SpellLink id={SPELLS.COMBO_BREAKER_BUFF.id} icon /> procs and not hitting all your <SpellLink id={SPELLS.FISTS_OF_FURY_CAST.id} icon /> ticks is a loss of potential damage.</Wrapper>,
+      requirements: () => {
+        return [
+          new Requirement({
+            name: <Wrapper>Wasted <SpellLink id={SPELLS.COMBO_BREAKER_BUFF.id} icon /> procs </Wrapper>,
+            check: () => this.comboBreaker.suggestionThresholds,
+          }),
+          new Requirement({
+            name: <Wrapper>Hit with all 5 ticks from <SpellLink id={SPELLS.FISTS_OF_FURY_CAST.id} icon /></Wrapper>,
+            check: () => this.fistsofFury.suggestionThresholds,
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: 'Use your cooldowns effectively',
+      description: <Wrapper>Your cooldowns have a big impact on your damage output. Make sure you use them as much as possible. <SpellLink id={SPELLS.TOUCH_OF_KARMA_CAST.id}icon/> is both a defensive and offensive cooldown, but is mostly used offensively.</Wrapper>,
+      requirements: () => {
+        const combatant = this.combatants.selected;
+        return [
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.STORM_EARTH_AND_FIRE_CAST,
+            when: !combatant.hasTalent(SPELLS.SERENITY_TALENT.id),
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.SERENITY_TALENT,
+            when: combatant.hasTalent(SPELLS.SERENITY_TALENT.id),
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.TOUCH_OF_DEATH,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.TOUCH_OF_KARMA_CAST,
+          }),
+          new Requirement({
+            name: <Wrapper> Absorb from <SpellLink id={SPELLS.TOUCH_OF_KARMA_CAST.id} icon /> used</Wrapper>,
+            check: () => this.touchOfKarma.suggestionThresholds,  
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.INVOKE_XUEN_THE_WHITE_TIGER_TALENT,
+            when: combatant.hasTalent(SPELLS.INVOKE_XUEN_THE_WHITE_TIGER_TALENT.id),
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: 'Manage your resources',
+      description: <Wrapper>Windwalker is heavily dependent on having enough Chi to cast your core spells on cooldown. Wasting Chi either by generating while capped or using <SpellLink id={SPELLS.BLACKOUT_KICK.id}icon/> and <SpellLink id={SPELLS.SPINNING_CRANE_KICK.id}icon/> too much will cause you to delay your hard hitting Chi spenders and lose damage.</Wrapper>,
```

Comment:
You mention using blackout kick too much is bad but none of the metrics track that. Did you forget to add it or is there just no module for tracking that?

Reply:
There is no metric, it's only really measurable by having low cast efficiency on cooldown Chi spenders. I have thought of looking at a "Chi spent on Blackout Kick vs other chi spenders ratio" but it's going to be tricky finding the correct ratio to measure by.

Reply:
maybe you could do something like Blackout kicks cast while other spenders were available? Anyway you don't to add it here just a thing for the future.